### DISCUSSION TITLE
how to show view page instead of listing page

### DISCUSSION THREAD

**Discussion by alisalehi1380** [filamentphp/filament#13475] [Help]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

I want the user to go to the `profileEdit page` instead of the `'index' => ListProfiles::route('/')` page when he clicks on the profile section in the menu. How can I implement it?

**Comment by leandrocfe** [comment]
You don't need to create a resource. Use a custom navigation item instead

https://filamentphp.com/docs/3.x/panels/navigation/#registering-custom-navigation-items

**Reply by alisalehi1380** [reply]
Yes, but I wanna show the listProfile page to the admin!
When the admin clicks on profiles, the list should be displayed to him.
But the user should not have access to the list page and see them.

**Reply by leandrocfe** [reply]
Maybe you can redirect users to the profileEdit page in the mount method

ListProfile
```php
public function mount(): void
{
    parent::mount();

    if (! auth()->user->is_admin) {
        //redirect...
    }
}
```
